src/js/edu-benefits/5490/config/form.js

```diff
@@ -254,8 +256,8 @@ const formConfig = {
                 'transferOfEntitlement',
                 'veteranFullName',
                 'view:veteranId',
-                'other',
-                'view:noPreviousBenefits'
+                'view:noPreviousBenefits',
```

Comment:
You're right. I confirmed with Alex the other day, just hadn't made the change yet.